models/models.go

```diff
@@ -25,11 +25,13 @@ const (
 )
 
 type StepBundleModel struct {
+	Inputs       []envmanModels.EnvironmentItemModel `json:"inputs,omitempty" yaml:"inputs,omitempty"`
```

Comment:
I see you reused the existing `EnvironmentItemModel`. I have introduced the same field for the graph pipeline workflow variant:
https://github.com/bitrise-io/bitrise/blob/a025238314dc80f594c64c9f1053d1f2791da056/models/models.go#L88

And I thought I will use a new type to indicate that these are not env vars. They might be implemented the same way underneath but they are more like "function parameters". 

What if we create a new type (like `BitriseInputModel`) and share it everywhere we will use the input field? I am not a 100% sure in it but it feels like this input field behave the same way everywhere.